avm/res/service-fabric/cluster/tests/e2e/max/main.test.bicep

```diff
@@ -74,15 +78,6 @@ module testDeployment '../../../main.bicep' = [
         clusterApplication: 'cf33fea8-b30f-424f-ab73-c48d99e0b222'
         tenantId: tenant().tenantId
       }
-      certificateCommonNames: {
```

Comment:
Would it hurt to have also the removed parameters used here? Asking since a max test intends to validate as many parameters as possible

Reply:
Add this parameter back to the test case, and removed the certificate param as both cannot be used together. At least one is required to satisfy the secure cluster requirement, otherwise the deployment will fail.

Reply:
Makes sense and probably explains why we had an additional certificate test. As long as we have at least one test covering each it's fine